safeweb/http.go

```diff
@@ -376,10 +337,15 @@ func (s *Server) serveBrowser(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Security-Policy", s.csp)
 	w.Header().Set("X-Content-Type-Options", "nosniff")
 	w.Header().Set("Referer-Policy", "same-origin")
-	if s.SecureContext {
+	if s.ServeHSTS {
 		w.Header().Set("Strict-Transport-Security", cmp.Or(s.StrictTransportSecurityOptions, DefaultStrictTransportSecurityOptions))
 	}
-	s.csrfProtect(s.BrowserMux).ServeHTTP(w, r)
+	if !slices.Contains([]string{"GET", "HEAD", "OPTIONS"}, r.Method) && r.Header.Get("Sec-Fetch-Site") != "same-origin" {
```

Comment:
should there be an option to allow `same-site`?

Reply:
I can't think of an internal case where we need it but we can def allow it via configuration :thumbsup: